src/components/SwitchView/index.js

```diff
@@ -20,6 +20,9 @@ type PropsShape = {
   views: Array<ViewItemShape>,
 };
 
+/**
+ * SwitchView is a 'tabbed' display that shows buttons for each configured 'view' and renders them when clicked or chosen as default
```

Comment:
what's our char limit on these? This line seems a bit longer than we typically expect it to be